bin/exclude

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""
+Creates a list of exclusions from the provided metadata TSV in the Nextstrain
+exclusions.txt convention. Prints the list to stdout.
+"""
+import os
+import json
+import logging
+import requests
+import argparse
+import pandas as pd
+from pathlib import Path
+from datetime import date
+from urllib3.exceptions import HTTPError
```

Comment:
Several of these imports aren't used.

Reply:
Ugh. Setting up python linting in VS Code is not very straightforward. I'm going to start using `pylint` at the command line.